gnss_msgs/msg/RtkAvailable.msg

```diff
@@ -0,0 +1,6 @@
+#if available and synchronized to ROS (UTC) time, time of actual sensor measurement
```

Comment:
You should be using https://github.com/Humatics/le_msgs/tree/rle_develop/gnss_msgs inside `le_msgs` so that projects outside this one can access this message definition. You need to open a branch HIT-244 inside `le_msgs` with this modification.

you should have le_msgs as a submodule to this repo, or have an analagous workspace similar to `location_engine` that contains both this repo and `le_msgs`.

Reply:
Addressed in https://github.com/Humatics/ethz_piksi_ros/pull/1/commits/d82302c181df7ea34a616402c5068391df69cda9

I have also opened a HIT-244 branch in ```le_msgs``` with these modifications